src/config.js

```diff
@@ -637,4 +637,18 @@ export default Object.freeze({
     VAULT_STATUS: "active",
     CHAIN_ID: 1,
   },
+  coinsonfire: {
+    TITLE: "Coins On Fire",
+    LOGO: "🪙🔥",
+    VAULT_ABI: yVaultV2,
+    VAULT_TYPE: "experimental",
+    VAULT_ADDR: "0xe7bf2905371cfeaeefe2d4c2c386b6b2a7e4270a",
```

Comment:
needs to be a 0.4.2 clone.

Reply:
this is an old one, we need to replace this with 0.4.2